docs/articles/tips-and-tricks/preferred-topology.rst

```diff
@@ -0,0 +1,69 @@
+Preferred topology
+===================
+
+If you want the flexibility to run tests against all topologies, but do not
```

Comment:
The preffered topology marker (``pytest.mark.preferred_topology``) can be used if you ...